packages/next/src/views/Document/index.tsx

```diff
@@ -96,7 +96,8 @@ export const Document: React.FC<AdminViewProps> = async ({
     DefaultView = collectionViews?.DefaultView
 
     if (!CustomView && !DefaultView) {
-      return <NotFoundClient />
+      const ErrorView = collectionViews?.ErrorView
```

Comment:
Actually yea, needs this:
 
 ```suggestion
      const ErrorViewToRender = collectionViews?.ErrorView || ErrorView
```